docs/data-management.rst

```diff
@@ -130,6 +130,33 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+
+Manage Images in Dataset/Projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can organize the data in the left-hand side tree by creating new Projects, Datasets and linking the Images to those new containers.
```

Comment:
Probably should be: "Projects *and* Datasets". If that's too many "and"s then maybe re-arrange?

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/5/commits/9ca345796046652263f447d141134c4210a317b3